src/screens/ContactScreen/index.js

```diff
@@ -36,7 +39,9 @@ const ContactScreen = ({navigation}) => {
   const [selectedUsers, setSelectedUsers] = React.useState([]);
   const [isSearchMode, setIsSearchMode] = React.useState(false);
   const [isLoadingSearchResult, setIsLoadingSearchResult] = React.useState(false);
-  const {createSignChat, loadingCreateChat} = useCreateChat();
+  const {createSignChat, createAnonymousChat, loadingCreateChat} = useCreateChat();
+  const route = useRoute();
+  const {from: sourceScreen} = route.params;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `route.params` object is being destructured to get the `from` property. However, there's no null/undefined check before accessing `params`. This could potentially lead to a TypeError if `params` is undefined.
```diff
- const {from: sourceScreen} = route.params;
+ const {from: sourceScreen} = route.params || {};
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->